addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,105 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as ts from 'typescript';
+
+interface MapType {
```

Comment:
I wanted to map relevant files with their relative path imports. So, it seems I can omit this declaration, store the results in an object.